### ISSUE TITLE
Release 2.3.1 wheel includes broken code

### ISSUE THREAD

**Issue by rhys-adams** [googleapis/python-api-core#323]
The `google_api_core-2.3.1-py2.py3-none-any.whl` file from PyPI includes what appears to be a patch failure:

```
    13059  12-15-2021 16:16   google/api_core/retry_BACKUP_3939795.py
    12781  12-15-2021 16:16   google/api_core/retry_BASE_3939795.py
    12770  12-15-2021 16:16   google/api_core/retry_LOCAL_3939795.py
    12786  12-15-2021 16:16   google/api_core/retry_REMOTE_3939795.py
```

This is causing my Cloud Function build to fail (log below).

#### Environment details

  - OS type and version: Google Cloud Build environment and local Linux (NixOS 21.05)
  - Python version: 3.9.9
  - pip version: 21.2.4
  - `google-api-core` version: 2.3.1

#### Steps to reproduce

  1. Try to deploy a Cloud Function with `google-api-core` in the dependency tree, eg. via a dependency on `google-cloud-aiplatform` in `requirements.txt`.
  2. Look Cloud Logs and see build failure.

#### Cloud Build log
```
Build failed: *** Error compiling './pip/google/api_core/retry_BACKUP_3939795.py'...
Sorry: IndentationError: expected an indented block (retry_BACKUP_3939795.py, line 204)
```

Current workaround is adding `google-api-core==2.3.0` to `requirements.txt`.

Ref #319.

**Comment by parthea** [comment]
Hi @rhys-adams,

Thank you for reporting this issue. Although I am able to install the wheels locally using `pip install *.whl`, I can see from the logs that you shared that the wheels is broken. I've published a new version 2.3.2. Please let me know if this fixes the issue!

**Comment by parthea** [comment]
I'm going to mark this issue as resolved as I've yanked the 2.3.1 release from PyPI. See the release history [here](https://pypi.org/project/google-api-core/#history).

Version 2.3.2 is available [here](https://pypi.org/manage/project/google-api-core/release/2.3.2/). 

Please re-open this issue if you're still having trouble with 2.3.2.
